Gemfile

```diff
@@ -18,7 +18,7 @@ gem "faraday-retry", "~> 2.2"
 gem "faraday-restrict-ip-addresses", "~> 0.3.0", require: "faraday/restrict_ip_addresses"
 gem "good_job", "~> 3.99"
 gem "gravtastic", "~> 3.2"
-gem "honeybadger", "~> 5.5.1", require: false # see https://github.com/rubygems/rubygems.org/pull/4598
+gem "honeybadger", "~> 5.26.2", require: false # see https://github.com/rubygems/rubygems.org/pull/4598
```

Comment:
Seems fixed, https://github.com/honeybadger-io/honeybadger-ruby/issues/537#issuecomment-2129947829, ok to remove the comment and try @segiddins?

Reply:
Can we merge and then remove the comment if it’s fixed?

Reply:
sounds like a plan